### PR TITLE
Fix multiple database connections - issue #86

### PR DESCRIPTION
**Pull request by Bernardino18**
Credits go to @azalty.
Tested on csgo game servers during 24 hours.

https://github.com/nuclearsilo583/zephyrus-store-preview-new-syntax/issues/86#issuecomment-1197568588

### CODE REVIEW

File: addons/sourcemod/scripting/store_item_cpsupport.sp [hunk 1]
```diff
@@ -214,6 +214,10 @@ public Action Command_TGS(int client, int args)
 
 public void Store_OnConfigExecuted(char[] prefix)
 {
+	// If it's already connected we are good to go
+	if(g_hDatabase != INVALID_HANDLE)
```
Comment (azalty) [inline addons/sourcemod/scripting/store_item_cpsupport.sp:218]: You can just do `if (g_hDatabase)` :)

**Review by azalty — CHANGES_REQUESTED** [review]
You should still let the ChatPrefix `strcopy` and `ReadCoreCFG()` functions be ran, so the tag updates properly if the store config changes

**Comment by nuclearsilo583** [comment]
thanks for fixing. I currently don't have much free time now.

**Comment by azalty** [comment]
@nuclearsilo583 I still believe that it's best to still update the store prefix and reload the configs on each map change

**Pull request merged**